query/graphql/pom.xml

```diff
@@ -147,7 +157,10 @@
                             lombok,
 
                             graphql-java-annotations,
-                            graphql-java-servlet
+                            graphql-java-servlet,
+
+                            jetty-servlets,
```

Comment:
Do we really need to embed these bundles? It looks like `jetty-servlets` is exporting the `org.eclipse.jetty.servlets` package.

Reply:
I was getting errors when I didn't let me try it again

Reply:
```java.lang.ClassNotFoundException: org.eclipse.jetty.servlets.DoSFilter cannot be found by admin-graphql_1.1.0.SNAPSHOT```